internal/injector/aspect/advice/call.go

```diff
@@ -40,13 +45,65 @@ func (a *appendArgs) Apply(ctx context.Context, chain *node.Chain, csor *dstutil
 		}
 	}
 
-	call.Args = append(call.Args, newArgs...)
+	if !call.Ellipsis {
+		call.Args = append(call.Args, newArgs...)
+		return true, nil
+	}
+
+	// The function call has an ellipsis, so we need to append our new arguments to the last argument,
+	// which is a slice. To do so, we need to provision a new slice of the right type and size, append
+	// all the relevant data in there, and then replace the last argument with the new slice.
+	lastIdx := len(call.Args) - 1
```

Comment:
what allows you to know that `len(call.Args) - 1 >= 0` so the following code never panics?

Reply:
The call has an ellipsis, so it has at least 1 argument... If not, the code includes a syntax error and won't compile anyway...